presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,452 @@ Except for the limit on queued queries, when a resource group runs out of a reso
+create a file named ``resource-groups.properties``. In this file, add a line that specifies the location of
+your JSON file: ``resource-groups.config-file = <file_path>``, where ``<file_path>`` is the path to your JSON file.
+
+Restart the Presto server: After creating the JSON file and pointing to it in the Presto configuration,
+restart your Presto server. The new resource groups will take effect after the restart.
+
+.. code-block:: text
 
     resource-groups.configuration-manager=file
-    resource-groups.config-file=etc/resource_groups.json
+    resource-groups.config-file=etc/resource-groups.json
+
+
+Database Resource Group Manager
+-------------------------------
+The Database Resource Group Manager in PrestoDB is a way to manage resources using a relational database.
+This allows dynamic changes to the resource group configurations without needing to restart
+the Presto server.
+
+Here's a brief guide on how to set it up:
+
+Set up a relational database: The database should be accessible by Presto.
+It will be used to store the configurations of your resource groups.
+
+Create tables for resource groups and selection rules: You need to create tables in the
+database that will store the definitions of your resource groups and the rules for selecting
+the appropriate resource group for a given query.
+
+Specify the database in the Presto configuration: In the ``etc`` directory of your Presto installation,
+create a file named ``resource-groups.properties``. In this file, add a line that specifies
+the JDBC URL of your database: ``resource-groups.config-db-url``=<jdbc_url>, where <jdbc_url> is the
+JDBC URL of your database.
+
+The database resource group manager loads the configuration from a relational database. The
+supported databases are MySQL, PostgreSQL, and Oracle.
+
+.. code-block:: text
+
+    resource-groups.configuration-manager=db
+    resource-groups.config-db-url=jdbc:mysql://localhost:3306/resource_groups
+    resource-groups.config-db-user=username
+    resource-groups.config-db-password=password
+
+Here's an example of what the tables in your database might look like:
+
+Resource Groups Table
+---------------------
+
+``resource_group_id`` : The ID of the resource group.
+name: The name of the resource group.
+
+``soft_memory_limit``: The maximum amount of memory that the resource group can use.
+max_queued: The maximum number of queued queries that the resource group can have.
+
+``hard_concurrency_limit``: The maximum number of queries that the resource group can run concurrently.
+
+Selectors Table
+-----------------
+
+``resource_group_id``: The ID of the resource group that this selector applies to.
+
+``user_regex``: A regex pattern that matches the user names for this selector.
+
+``source_regex``: A regex pattern that matches the source names for this selector.
+
+With the Database Resource Group Manager, changes to the configuration in the database
```

Comment:
This seems to repeat the content of lines 109-111, the first paragraph after the heading "Database Resource Group Manager". I suggest you delete this paragraph. 

But I like the way you describe it here. So I suggest that before you delete it here, compare these words to 109-111. Consider revising 109-111 with some or all of the words here. 
Decide which version best describes what you want to say and use the first version in 109-111, the second version here, or a combination of the two.